website/docs/Guides-Typescript/development.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 10
+title: Development
+---
+
+Development relies on the following tools being installed:
+
+* Node.js: JavaScript runtime
+* NPM (bundled with Node.js)
+* (optional) PlantUML. An IDE plugin such as the one for IntelliJ IDEA also does the trick.
+
+Note that development tends to be done against the latest LTS (or even more recent) versions of Node.js and NPM.
+
+## Main commands
+
+Run the following command to setup the project:
+
+```shell
+npm run clean
+npm install
+npm run setup
+```
+Run the following command to **build** each of the packages:
+
+```shell
+# Build the project
+npm run build
+```
+
+This includes cleaning up and installing any NPM (dev) dependencies.
+
+The preceding commands can also be run as follows:
+
+```shell
+npm run initialize
+```
+
+The following command statically _style_-checks the source code in all the packages:
+
+```shell
+# Run lint
+npm run lint
+```
+
+*Note* that this does not catch TypeScript compilation errors!
+(That's because linting only does parsing, not full compilation.)
+
+Run the following command to run the tests:
+
+```shell
+# Run the tests
+npm run test
```

Comment:
"`npm test`" does the same thing, and is slightly more ideomatic.

Reply:
noted, thank you